templates/root/.subversion/auth/svn.ssl.server/7572ab03ba0fa67798aca4e124c6dc9f

```diff
@@ -0,0 +1,13 @@
+K 10
+ascii_cert
+V 1852
+MIIFaDCCA1ACCQCX8YA7MWXZZDANBgkqhkiG9w0BAQsFADB2MQswCQYDVQQGEwJVUzELMAkGA1UECAwCTlkxETAPBgNVBAcMCE5ldyBZb3JrMSIwIAYDVQQKDBlPcGVuIFRlY2ggU3RyYXRlZ2llcywgTExDMSMwIQYDVQQDDBpzdm4ub3BlbnRlY2hzdHJhdGVnaWVzLmNvbTAeFw0xNjA4MTUyMDA4MDBaFw0yNjA4MTMyMDA4MDBaMHYxCzAJBgNVBAYTAlVTMQswCQYDVQQIDAJOWTERMA8GA1UEBwwITmV3IFlvcmsxIjAgBgNVBAoMGU9wZW4gVGVjaCBTdHJhdGVnaWVzLCBMTEMxIzAhBgNVBAMMGnN2bi5vcGVudGVjaHN0cmF0ZWdpZXMuY29tMIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAo5t7ms2ipDlwqw8xJy6P03P53FMtnmcoJDN3JKRJiSEPuRdbwkuch7nfqn9vsuYDrN/klnXysH/xYuECE0M7KPbnMx/M2rBvANadfwbTUcYKyGzUe8fVz/yyNgyH7S+YiO7WCJH2UDYM0AY/omLAafNJfMeibZCembssQ+QTBRvvi1ZypOqOevDzPWmGqBZCtgOcEOMP39ktb24GYPUsK4YSuo+qMiX+oDntIbWbhOGlt14t36c58VauKc8FQmdmZMPG1Io9RkfV2B2x2lD64LbDaRSQpRUuuAmJ1TjXBBO+mYrty0bon3ZjoQI7wUZtzQikEUc4F2Us0AVupHsPbjB3dgXw18XKXzOwAz51vYWamI8iotxvMemjtkcDRdeWbXWMh+AYnd6TPcglSxMQAtp7TGcfJEaX54TQqrBdJ6OFguG5djUAIGnS/W77wB9yM+Vw59geDDVneuRlJn/QPD5iBdgMPZgKkDDAViIjEDaRGCGIcp7kuajz1iz/usJ/VlQmjWlKOJ0CAHdHjbj6pdnnYr292kKhX0jARdUDUKFR/qrZRMZhQxUnQEr212xfrTJLV5gKJ5NY0OmsstYR0iA2QvLariO9EPEqM6CAKmhKWj47oX8y95NIngiv1xPBBQOy3FRaKbekrgX9TNjEV1iYWqVnUm3bpEktNzdc0x0CAwEAATANBgkqhkiG9w0BAQsFAAOCAgEACg8mHo2O7bQ+gndZpt5PI4175dUv9SWnRrLfVmVmNmYBRLclB931e5i86Tp9iQ0uiLLdwi1SaZ0ZrPixtc4EfGYlyHkLsTB7lfXNQxqt7nSs+KaUS8N7N+k6RTpakrQqgPsdQdd/4/t8F+lY+mebO17iY3gmjyjxCq7fEZW/5ZCjHZt6sSkapo5dDUlQBjX2CsqGYEDABpoHnyJ707cJffraq+vL/O5KgBUtavY7oAt31wf48dHSHDCCABGMVUwEz6nkK9kNXakWS1E+a0PVMruaFGbYGw7S7AjKu+TJKAbYBIaki5xbFMIARFfY6vn8iL/IRiGGRg/X7h4bID33241XSerdXgxAUssF74hYQHGLJ1ZZQ688OOOoJdJwicvgJLfiMZdxGJXMYQjW7+fLqI8y5IrrZQHlk/+ABANxXPpxmNJ1Om6PdS2jhnG+O9ab3UqERhQPRFiOEUQ+gfPIZMaantcwzBZapdWREnYt1X7syG5B6lk1j04aJO31Y9VqZsJp43aSoowQRTOre6HEF/1HtZx80eoHcZ+rraxcpfcg6rfQJoWw/r2TNXKcHXJQzXc22RPfOcXY82JTEt9bfUWnxoEXzT2SAKCR/P2qfZq+OrwcoEj+8TKCMF0mBVpPyzXnVuRGWEQjBMfeCgDsMzqd9TNisr1ur04JgicJ6OU=
```

Comment:
Typo: the 122nd letter in the cert string should be lower-case "b" not upper case "B".

**JUST KIDDING**